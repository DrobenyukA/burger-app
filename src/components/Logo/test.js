import React from 'react';
import renderer from 'react-test-renderer';
import Logo from './Logo';

describe("<Logo />", () => {
    it("Capturing Logo`s snapshot", () => {
        const renderedComponent = renderer.create(<Logo classes="test-logo" height="90" />).toJSON();
        expect(renderedComponent).toMatchSnapshot();
    })
});
