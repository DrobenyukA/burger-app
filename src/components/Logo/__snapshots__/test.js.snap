// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> Capturing Logo\`s snapshot 1`] = `
<div
  className={undefined}
  style={
    Object {
      "height": "90",
    }
  }
>
  <img
    alt="MyBurger"
    src="burger-logo.png"
  />
</div>
`;
